data_structures/queue_with_stacks/queue_with_stacks.py

```diff
@@ -0,0 +1,34 @@
+from .stack import Stack
+
+
+class Queue:
+    """class for Queue"""
+    def __init__(self, iter=[]):
+        self.stack1 = Stack()
+        self.stack2 = Stack()
+        self._len = 0
+
+        for item in iter:
+            self.enqueue(item)
+
+    def enqueue(self, val):
+        """add new iten to the queue"""
+        if val:
```

Comment:
This disallows enqueueing any value that is false-y. However, you should allow the user to add any value they want.